pos_print_sale_order/static/src/js/SaleOrderManagementScreen.js

```diff
@@ -0,0 +1,388 @@
+odoo.define("pos_print_sale_order.PrintSaleOrderManagementScreen", function (require) {
+    "use strict";
+
+    const SaleOrderManagementScreen = require("pos_sale.SaleOrderManagementScreen");
+    const Registries = require("point_of_sale.Registries");
+    const {sprintf} = require("web.utils");
+    const {parse} = require("web.field_utils");
+    const {Orderline} = require("point_of_sale.models");
+
+    /**
+     * ID getter to take into account falsy many2one value.
+     * @param {[id: number, display_name: string] | false} fieldVal many2one field value
+     * @returns {number | false}
+     */
+    function getId(fieldVal) {
+        return fieldVal && fieldVal[0];
+    }
+
+    const PrintSaleOrderManagementScreen = (SaleOrderManagementScreen) =>
+        class PrintSaleOrderManagementScreen extends SaleOrderManagementScreen {
+            /**
+             * @override
+             */
+            async _onClickSaleOrder(event) {
+                const clickedOrder = event.detail;
+                const {confirmed, payload: selectedOption} = await this.showPopup(
```

Comment:
not a good practice, but perhaps a better solution is to change the showPopup method to add additional items to the list instead of completely copying the current method.

e.g.:

```js
async _onClickSaleOrder(event) {
    this.additionalSelectionList = [{id: "2", label: this.env._t("Print"), item: "print"}];
    await super._onClickSaleOrder(...arguments);
    this.additionalSelectionList = false;
}
showPopup(name, props) {
    if (name == "SelectionPopup" &&  this.additionalSelectionList) {
        props = [...props, ...this.additionalSelectionList];
    }
    // TODO: add calback after super for print item of selection 
    return super.showPopup(name, props);
}
```
what do you think?